praw/reddit.py

```diff
@@ -588,7 +588,7 @@ def _objectify_request(
         )
 
     def _handle_rate_limit(
-        self, exception: RedditAPIException
+        self, exception: "RedditAPIException"
```

Comment:
Why did this need to be quoted? Did it not work before when un-quoted?

Reply:
I don't know, I think this is another case of accidental quotation but I might be wrong.

Reply:
```suggestion
        self, exception: RedditAPIException
```